### ISSUE TITLE
Images fail to load due to Invalid 'blob' protocol 

### ISSUE THREAD

**Issue by skyghis** [tiliado/nuvolaplayer#145]
## Issue

A recent update break image download (cover, user profile image, ...) in Plex.
The result is an interface with only place-holder images.

## Log extract

```
[Worker:WARNING  Nuvola] WebExtension.vala:295: Invalid URI protocol 'blob:http://plex.skyghis.me:32400/eb687627-e7f3-4d4d-b97b-be8903a76082'.
[Runner:WARNING  Nuvola] AppRunnerController.vala:641: Communication failed: Failed to connect to socket '/tmp/libdiorite.nuvolaplayer3.plex.webworker(skyghis)'. Connection refused
[Runner:WARNING  Nuvola] AppRunnerController.vala:641: Communication failed: Failed to connect to socket '/tmp/libdiorite.nuvolaplayer3.plex.webworker(skyghis)'. Connection refused
[...]
[Worker:WARNING  Nuvola] WebExtension.vala:295: Invalid URI protocol 'blob:http://plex.skyghis.me:32400/a4083319-713e-4b82-89ad-ca80332caa2f'.
[Worker:WARNING  Nuvola] WebExtension.vala:295: Invalid URI protocol 'blob:http://plex.skyghis.me:32400/2308c857-3513-47ba-a88b-4ea2688774c0'.
[...]
```

Sample of html code causing the URI error:
```html
<div style="background-image: url(&quot;blob:http://plex.skyghis.me:32400/b6c7b622-5810-4431-a1bd-a9f87b2eb69e&quot;);" class="media-poster">
 [...]
</div>
```

## System

* Nuvola Player 3 Beta version 3.0.0+201508211407.2fd2707 [From launchpad repos](http://ppa.launchpad.net/numix/ppa/ubuntu)
* Using Plex Music version 1.1
* Ubuntu 15.04 (Unity)
* Kernel 3.19.0-26-generic x86_64

**Comment by fenryxo** [comment]
Thanks for your bug report. Fixed in git master & 3.0.x branches. Will be available with the next rebuilt of packages.

**Comment by skyghis** [comment]
Fix confirmed in version 3.0.0+201508311518.b1fe746
Thanks you
